takwimu/takwimu_ui/src/Theme.js

```diff
@@ -12,6 +12,18 @@ const Theme = createMuiTheme({
   },
   typography: {
     fontHeading: FONT_FAMILY_HEADING,
+    fontText: FONT_FAMILY_TEXT,
+    h1: {
```

Comment:
`h1` is for the main title in the hero section... `h2` already has the styling you need for this component.